src/lib/example.js

```diff
@@ -1,13 +1,19 @@
 import chalk from 'chalk';
 import enquirer from 'enquirer';
 import fs from 'fs-extra';
+import path from 'node:path';
 import url from 'node:url';
+import util from 'node:util';
 import ora from 'ora';
-import path from 'path';
 import shell from 'shelljs';
-import util from 'util';
 import Constants from './constants.js';
-import { setupProject } from './helpers.js';
+import { isDirEmpty, setProjectName, setupProject, step } from './helpers.js';
+
+// Public API
+export default example;
```

Comment:
Same comment about Public and Private API label.

Reply:
Let's discuss it in the comment [above](https://github.com/o1-labs/zkapp-cli/pull/661#discussion_r1681402352) (should be applied to all files).